src/Editor/index.js

```diff
@@ -10,7 +10,7 @@ import {
     TouchableOpacity
 } from 'react-native';
 import XRegExp from 'xregexp'
-import EU from './EditorUtils';
+import EU, {formattedTextToPlain} from './EditorUtils';
```

Comment:
Don't import `formattedTextToPlain`, just use it as `EU.formattedTextToPlain` as the rest of the other functions.

Reply:
Will move to EU